genenetwork/genenetwork2
# Added scroller to trait page and changed sample tables to use it

## Description
This PR will add Scroller functionality to some GN tables and address the various form submissions/javascript issues caused by it. Since all table rows (for example sample values in the show_trait page) won't be loaded in the DOM at once anymore (since Scroller loads them as you scroll), the way data is fetched/change will need to be changed to use DataTables' API.

#### How should this be tested?
Load a trait page (or any other page) with a large table, and it should load faster. For example - http://genenetwork.org/show_trait?trait_id=10007&dataset=HSNIH-PalmerPublish

#### Any background context you want to provide?
We've had problems for a while with large tables taking a long time to load. I looked at Scroller once in the past, but I didn't realize at the time that there was a way to address the issue of all rows not being directly accessible by javascript/form submission.

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions
N/A
